python/lsst/ap/verify/ingestion.py

```diff
@@ -303,15 +293,11 @@ def _ingestDefects(self, dataset, workspace):
         if os.path.exists(os.path.join(workspace.calibRepo, "defects")):
             self.log.info("Defects were previously ingested, skipping...")
         else:
-            if self.config.defectTarball:
-                self.log.info("Ingesting defects...")
-                defectFile = os.path.join(dataset.defectLocation, self.config.defectTarball)
-                self._doIngestDefects(workspace.dataRepo, workspace.calibRepo, defectFile)
-                self.log.info("Defects are now ingested in {0}".format(workspace.calibRepo))
-            else:
-                self.log.info("No defects to ingest, skipping...")
+            self.log.info("Ingesting defects...")
+            self._doIngestDefects(workspace.dataRepo, workspace.calibRepo, dataset.defectLocation)
+            self.log.info("Defects are now ingested in {0}".format(workspace.calibRepo))
```

Comment:
Does there need to be a way to disable defect ingestion for instruments that don't use it (e.g., HSC), or is that something `IngestDefectsTask` takes care of?

Reply:
It looks like the changes in obs_decam result in ingested defects always landing in obs_decam_data. Do we even need to deal with defects in ap_verify **at all** anymore?

Reply:
They do not land in `obs_decam_data`.  They land in `workspace.calibRepo`.

They will need to be ingested in order to use them.